cmd/exo/cmd/root.go

```diff
@@ -229,6 +229,9 @@ func initConfig() {
 	if gCurrentAccount.DNSEndpoint == "" {
 		gCurrentAccount.DNSEndpoint = strings.Replace(gCurrentAccount.Endpoint, "/compute", "/dns", 1)
 	}
+
+	gCurrentAccount.Endpoint = strings.TrimRight(gCurrentAccount.Endpoint, "/")
```

Comment:
I think you have to put TrimRight() before Replace()

Reply:
`gCurrentAccount.Endpoint` cannot be empty at this point.